pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -234,9 +234,8 @@ class Dialog extends React.Component {
                 </Button>
             </Popover>;
 
-        const error = this.props.error || this.props.static_error;
-        const error_alert = error &&
-            <Alert variant='danger' isInline title={React.isValidElement(error) ? error : error.toString() }>{error.details}</Alert>;
+        const error_alert = this.props.error &&
+            <Alert variant='danger' isInline title={this.props.error} />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19505-20231019-075004-e634b41b-fedora-38-devel-storage//Coverage/lcov/github-pr.diff.gcov.html)